### DISCUSSION TITLE
wanted to use go-re2 in corazawaf

### DISCUSSION THREAD

**Discussion by joshi-mohit** [wasilibs/go-re2#59] [General]
Facing some performance issues in regex match in coraza ...How can I use this in corazawaf .. I have an application that uses corawaf .. and now on corazawaf I want to use this in say file internal/corazawaf/waf.go as a replacement of regexp

**Comment by anuraaga** [comment]
Hi @joshi-mohit - coraza provides a plugin using this library. Hopefully it helps.

https://github.com/corazawaf/coraza-wasilibs

**Reply by joshi-mohit** [reply]
Thanks a lot Anuraga, that works for me. I am still to run and take the numbers, will keep you posted
